src/main/scala/io/iohk/ethereum/ledger/BlockPreparator.scala

```diff
@@ -45,26 +46,47 @@ class BlockPreparator(
    * @return
    */
   private[ledger] def payBlockReward(block: Block, worldStateProxy: InMemoryWorldStateProxy): InMemoryWorldStateProxy = {
```

Comment:
Just minor comment: I would update scaladoc, to reflect all the changes and maybe mention treasury spec.

Reply:
Just did! I'm not sure whether to mention the spec or not as it's not public yet. I think the description I added should be a good enough description without having to refer to it